pandas/core/generic.py

```diff
@@ -7764,7 +7777,7 @@ def asfreq(
 
         >>> index = pd.date_range('1/1/2000', periods=4, freq='T')
         >>> series = pd.Series([0.0, None, 2.0, 3.0], index=index)
-        >>> df = pd.DataFrame({'s':series})
+        >>> df = pd.DataFrame({{'s':series}})
```

Comment:
`'s':series` -> `"s": series`?

Reply:
Thanks, I'll add the space; but all of the other examples use single quotes, so I'd stick to that.

Reply:
Fixed in 9c91ee7.